provider-utils/livestream-questions.json

```diff
@@ -270,9 +274,19 @@
               "type": "confirm",
               "question": "Do you want to modify any advanced video encoding parameters?",
               "required": true
-          }
+            },
+            {
+                "key": "mp4URL",
+                "question": "Provide URL to the MP4 file: ",
+                "validation": {
+                    "operator": "regex",
+                    "value": "^[a-z0-9_:\\\/]+$",
+                    "onErrorMsg": "mp4URL can only be a valid url"
```

Comment:
You can make error message more user friendly. An example: `Error the provided URL is not valid`

Reply:
Fixed the error message.